dev_guide/templates.adoc

```diff
@@ -122,8 +121,9 @@ uploading the template by piping the process output to the `create` command:
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v
-ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc create -f -
+$ osc process -f examples/sample-app/application-template-dockerbuild.json \
+    -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin \
+    | osc create -f -
```

Comment:
@CowboysFan Are there any guidelines on horizontal scrolling use?  (Preferably, along the lines of: "Don't do it -- readers will hate you!"  :smile: )